stdlib/xslt/src/main/ballerina/src/xslt/natives.bal

```diff
@@ -14,9 +14,14 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerinax/java;
+
 # Transforms the single-rooted XML content to another XML/HTML/plain text using XSL transformations.
 #
 # + input - An XML object, which needs to be transformed
 # + xsl - The XSL style sheet represented in an XML object
 # + return - The transformed result represented in an XML object or the `error` in case of an error
-public function transform(xml input, xml xsl) returns xml|error = external;
+public function transform(xml input, xml xsl) returns xml|error =
```

Comment:
Shall we create a separate __*private*__ inter-op function, and call it inside the __*public*__ function ?

Reply:
Why we need to create intermediated  function?

Reply:
It is the practice we had during inter-op migration. The intention is to keep the public function as a pure ballerina function.

Reply:
IINM,  we had to use intermediated function if this migration causing any API change due to that. Specially arguments with string types, we had to use it. But if exisiting API can support without breaking the backwards compatibility the we don't need another function to bridge IMO.

Reply:
+1.